_data/simple-icons.json

```diff
@@ -7465,6 +7465,16 @@
             "hex": "09009B",
             "source": "https://www.goindigo.in"
         },
+        {
+            "title": "Inductive Automation",
+            "hex": "435466",
```

Comment:
I'm not seeing this hex value anywhere on https://inductiveautomation.frontify.com/d/PvUNchzTCkyj/inductive-automation-brand-guidelines?#/basics/colors. I'd go for IA green maybe?

Reply:
I chose this color off the "one color" logo on this page:
https://inductiveautomation.frontify.com/d/PvUNchzTCkyj/inductive-automation-brand-guidelines#/basics/logos
Which I'm pretty confident is our preferred branding in monochrome environments.

Reply:
Huh, yeah, you're right. In my defense, the single color gear logo I used as a starting point _does_ use #435366 - weird. I'll let our branding team know about the discrepancy.